onnxruntime/python/tools/transformers/onnx_exporter.py

```diff
@@ -16,7 +16,7 @@
 from huggingface_models import MODEL_CLASSES
 from quantize_helper import QuantizeHelper
 from torch_onnx_export_helper import torch_onnx_export
-from transformers import AutoConfig, AutoTokenizer, LxmertConfig, TransfoXLConfig
+from transformers import AutoConfig, AutoModelForImageClassification, AutoTokenizer, LxmertConfig, TransfoXLConfig
```

Comment:
I suggest to use optimum package to export onnx of vit instead of adding it to onnx_exporter.py. See example here:
https://www.philschmid.de/optimizing-vision-transformer

Reply:
Sounds good, as discussed in the meeting I'll do a separate PR to unify all model exporting to use Optimum.